internal/client/consensus/grandpa/aux_schema_test.go

```diff
@@ -0,0 +1,376 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
```

Comment:
This file already existed right? Why doesnt it just show up as a moved file?

Reply:
I think cause I renamed this file.